project_stages.md

```diff
@@ -0,0 +1,176 @@
+
+#### Sandbox Entry Requirements and Considerations
+
+* Projects must have a minimum of two maintainers with different company affiliations.
+* Projects must be aligned with the OpenSSF mission and be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+
+
+#### Submission Forms for Sandbox
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+
+### Incubating
+
+![Incubating]
+
+Incubating projects represent maturing but not fully realized projects. Incubating projects have an established technical vision and working project, but are still refining implementation, discovering (and adapting to) new use cases, and operating as an open source team. 
+
+#### Project Responsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports
+* Implement, practice, and refine mature software development and release practices such a following a version schema
+
+#### Project Support
+* Receives guidance on technical direction from TAC
+* Receive a TAC or WG sponsor for guidance as an Incubation project
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may not be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are an "OpenSSF Incubating Project."  
+
+#### Incubating Entry Requirements and Considerations
+
+* Projects must have a minimum of three maintainers with a minimum of two different company affiliations.
+* Projects must be aligned with the OpenSSF mission and be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
```

Comment:
Pointing out the "address an unfulfilled need" phrase, as this is worth a TAC discussion.

Incentivizing collaboration is good, but as worded, it would seem to impose a requirement on projects that "there can only be one X, and the first one wins". Do we want that?

Reply:
+1 on the need for discussion. "Novel approach for existing areas" was an attempt to recognize that the first to apply isn't necessarily the only way (and disincentive rushed applications) , but there should be some mitigation to avoid duplicate functionality projects. I think given the OpenSSF's focus on improving existing project security, more resources for fewer projects is appropriate, rather than a market-driven approach.

Reply:
Correct!

Reply:
I'll propose an edit

Reply:
```suggestion
* Projects must be aligned with the OpenSSF mission _and_ either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
```

Reply:
LGTM!